resources/lang/ko.js

```diff
@@ -0,0 +1,165 @@
+      FoeRequiem: '마인의 진혼곡', // up 0x8b, down 0x8c
+    });
+
+    this.kUIStrings = Object.freeze({
+      // jobs: text on the pull countdown.
+      Pull: 'Pull',
+    });
+
+    // Due to this bug: https://github.com/ravahn/FFXIV_ACT_Plugin/issues/100
+    // We can not look for log messages from FFXIV "You use X" here. Instead we
+    // look for the actual ability usage provided by the XIV plugin.
+    // Also, the networked parse info is given much quicker than the lines from the game.
+    this.youUseAbilityRegex = function() {
+      var ids = this.AbilitiesToIds.apply(this, arguments);
+      return Regexes.Parse(' 1[56]:\\y{ObjectId}:' + this.playerName + ':' + Regexes.AnyOf(ids) + ':');
+    };
+    this.youStartUsingRegex = function() {
+      var ids = this.AbilitiesToIds.apply(this, arguments);
+      return Regexes.Parse(' 14:' + Regexes.AnyOf(ids) + ':' + this.playerName + ' starts using ');
+    };
+    this.youGainEffectRegex = function() {
+      var effects = [];
+      for (var i = 0; i < arguments.length; ++i) {
+        var effect = arguments[i];
+        this.ValidateEffect(effect);
+        effects.push(effect);
+      }
+      return Regexes.Parse(' 1A:' + this.playerName + ' gains the effect of ' + Regexes.AnyOf(effects) + ' from .* for (\\y{Float}) Seconds\.');
+    };
+    this.youLoseEffectRegex = function() {
+      var effects = [];
+      for (var i = 0; i < arguments.length; ++i) {
+        var effect = arguments[i];
+        this.ValidateEffect(effect);
+        effects.push(effect);
+      }
+      return Regexes.Parse(' 1E:' + this.playerName + ' loses the effect of ' + Regexes.AnyOf(effects) + ' from .*\.');
+    };
+
+    this.abilityRegex = function(abilityName, attacker, target, flags) {
+      this.ValidateAbility(abilityName);
+      if (!attacker)
+        attacker = '[^:]*';
+      // type:attackerId:attackerName:abilityId:abilityName:targetId:targetName:flags:
+      var r = ' 1[56]:\\y{ObjectId}:' + attacker + ':' + this.kAbilNameToId[abilityName] + ':';
+      if (target || flags) {
+        if (!target)
+          target = '[^:]*';
+        if (!flags)
+          flags = '[^:]*';
+        r += '[^:]*:\\y{ObjectId}:' + target + ':' + flags + ':';
+      }
+      return Regexes.Parse(r);
+    };
+
+    this.gainsEffectRegex = function(effect, target, attacker) {
+      this.ValidateEffect(effect);
+      if (!target)
+        target = '[^:]*';
+      if (!attacker)
+        attacker = '[^:]*';
+      return Regexes.Parse(' 1A:' + target + ' gains the effect of ' + effect + ' from ' + attacker + ' for (\\y{Float}) Seconds\.');
+    };
+    this.losesEffectRegex = function(effect, target, attacker) {
+      this.ValidateEffect(effect);
+      if (!target)
+        target = '[^:]*';
+      if (!attacker)
+        attacker = '[^:]*';
+      return Regexes.Parse(' 1E:' + target + ' loses the effect of ' + effect + ' from ' + attacker + '.*\.');
+    };
+
+    this.countdownStartRegex = function() {
```

Comment:
Are these the same in Korean? Could you add a "TODO: not translated yet" if not?

Reply:
![2017-12-27 16 14 06](https://user-images.githubusercontent.com/5380174/34373861-0b6b5dcc-eb21-11e7-8a8b-e890cef97a71.png)

Seems only 'gains/loses' and few things are in English, so this doesn't need to translate.